src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -99,7 +106,7 @@ public RestoreCommand()
                     int restoreCount = restoreInputs.V3RestoreFiles.Count;
 
                     int currentFileIndex = 0;
-
+                    bool printPackageSources = true;
```

Comment:
Moved the call to GetPackageSources outside of the loop in https://github.com/NuGet/NuGet.Client/commit/134f1f6bce281390b94e13132456f9913047e374